Added LF (L12L) modem details to the website

manuignatius opened the pull request:
Live website in staging: https://subnero.com/website-staging/

Please note that not all the links will work in staging website as it links back to the main website. The changed pages and features are as follows (they will work properly once we merge the PR):

- Landing page dropdown menu for products
- Product launch pulse page: https://subnero.com/website-staging/wnc/news/2024/10/05/Product-Launch-L12L-series-modems.html
- New button for L12L series in products page: https://subnero.com/website-staging/products/
- L12L series modems in modems page: https://subnero.com/website-staging/products/modem.html?&section=L12Lseries
- L12L SC: https://subnero.com/website-staging/products/wnc-l12lss4.html
- L12L OC: https://subnero.com/website-staging/products/wnc-l12lso4.html
- Product brochure: https://subnero.com/website-staging/brochures/Subnero-LF-Modems.pdf

I plan to make the website live after 30 October or after we get approval from relevant entities.

manuignatius committed:
WIP: LF modem web pages

manuignatius committed:
removing references to SWAN

manuignatius committed:
Updated images

manuignatius committed:
Updated images

manuignatius committed:
Reversed text order

jjophi committed:
Products wnc page update

jjophi committed:
Product WNC page update

jjophi committed:
Navigation submenu style

manuignatius committed:
Updated features to add JANUS support

manuignatius committed:
WIP: Pulse post

jjophi committed:
Submenu capitalization removed

manuignatius committed:
Added missing images

manuignatius committed:
Updated swis image

notthetup commented on products/wnc-l12lso4.md:
```diff
@@ -0,0 +1,120 @@
+
+## Silver Edition OEM Configuration Modem
+
+Offering communication ranges exceeding 10 km or up to 8 kbps of data rate, the L12L series of acoustic smart modems are designed for long-range subsea wireless communications in challenging environments. With features like an ultra-low power deep sleep state consuming less than 1 mW, precise localization and ranging, and arbitrary waveform transmission and recording capabilities, these modems ar ...
```
```suggestion
Offering communication ranges exceeding 10 km and up to 8 kbps of data rate, the L12L series of acoustic smart modems are designed for long-range subsea wireless communications in challenging environments. With features like an ultra-low power deep sleep state consuming less than 1 mW, precise localization and ranging, and arbitrary waveform transmission and recording capabilities, these modems are optimized for high-efficiency underwater operations.
```

notthetup commented on products/wnc-l12lss4.md:
```diff
@@ -0,0 +1,119 @@
+
+## Silver Edition Standalone Configuration Modem
+
+Offering communication ranges exceeding 10 km or up to 8 kbps of data rate, the L12L series of acoustic smart modems are designed for long-range subsea wireless communications in challenging environments. With features like an ultra-low power deep sleep state consuming less than 1 mW, precise localization and ranging, and arbitrary waveform transmission and recording capabilities, these modems ar ...
```
```suggestion
Offering communication ranges exceeding 10 km and up to 8 kbps of data rate, the L12L series of acoustic smart modems are designed for long-range subsea wireless communications in challenging environments. With features like an ultra-low power deep sleep state consuming less than 1 mW, precise localization and ranging, and arbitrary waveform transmission and recording capabilities, these modems are optimized for high-efficiency underwater operations.
```

mchitre reviewed (dismissed):
- Don't know what "optimum range and datarate" means. Optimum does NOT mean a good balance, but "best by some specific criteria".
- We do support BCH codes too, although perhaps not critical to highlight.
-

manuignatius committed:
Update products/wnc-l12lso4.md

Co-authored-by: Chinmay Pendharkar <553140+notthetup@users.noreply.github.com>

manuignatius committed:
Update products/wnc-l12lss4.md

Co-authored-by: Chinmay Pendharkar <553140+notthetup@users.noreply.github.com>

manuignatius committed:
Added BCH code to tech spec

manuignatius committed:
Updated brochure to add BCH code

manuignatius commented:
> * Don't know what "optimum range and datarate" means. Optimum does NOT mean a good balance, but "best by some specific criteria".

Cant find where I have used "optimum range and datarate" to update this.

> * We do support BCH codes too, although perhaps not critical to highlight.

Updated the website and all brochures to include all 3 FEC the we support: LDCP, BCH and convolution codes

mchitre commented:
> Cant find where I have used "optimum range and datarate" to update this.

`wnc-bands.png`

mchitre reviewed (dismissed):
OK other than "optimum"

manuignatius commented:
Ah, thats I was not able to find it using a search. 

I have updated it to "For the best overall performance" and also updated "optimum" to "best"

![wnc-bands](https://github.com/user-attachments/assets/f1f197a0-6ff6-48a3-bf32-167949f37157)
![wnc-edition](https://github.com/user-attachments/assets/754fc4e8-14eb-47ec-937f-dbfdfec65982)

manuignatius committed:
Updated images
